src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -227,7 +234,13 @@ const connectQueryRules: QueryRulesConnector = (render, unmount = noop) => {
         );
       },
 
-      render({ results, instantSearchInstance }) {
+      render({
+        results,
+        instantSearchInstance,
+      }: {
+        results: SearchResults;
```

Comment:
It's necessary at this point to add the definition again to remove the `undefined` type to access values.

Reply:
I don't understand why because `results`, `instantSearchInstance` are required at this point.

Reply:
Now that we made them required, no need for this!